test/unit/services/test-get-export-csv.js

```diff
@@ -37,17 +37,17 @@ describe('services/get-export-csv', function () {
     })
   })
   describe('should format the capacity when exporting overviews', function () {
-    it('should be formatted to two decimal figures', function () {
+    it('to two decimal figures', function () {
       var capacityExport = getExportCsv(orgUnit.TEAM.name, helper.TEAM_OVERVIEW_RESULT, tabs.OVERVIEW).csv
       expect(capacityExport).to.include('107.37')
```

Comment:
Since this is an include, it doesn't fully test that it is rounded to 2 decimal places, as a value of `107.37389748329` would still pass here as it includes `107.37`. Including the `%` symbol should sort this